documentation/cobigen-openapiplugin.asciidoc

```diff
@@ -40,18 +40,18 @@ Additionally, the java plugin provides the ability to match packages (`container
 
 * type 'element'
 
-The container matcher matches elements as Java Objects, in this case will be always an `OpenApiFile` object. (See link:cobigen-core_configuration#containermatcher-node[`containerMatcher` semantics] to get more information about `containerMatchers` itself.)
+The container matcher matches elements as Java Objects, in this case will be always an `OpenAPIFile` object. (See link:cobigen-core_configuration#containermatcher-node[`containerMatcher` semantics] to get more information about `containerMatchers` itself.)
 
-[[openapi-variable-assignment]]
+[[OpenAPI-variable-assignment]]
```

Comment:
this is a reference, you should not change here anything, please check all occurrences

Reply:
reverted changes in all occurences